chsdi/templates/htmlpopup/flugplaetze_heliports.mako

```diff
@@ -5,7 +5,7 @@
     <tr><td class="cell-left">${_('ch.bazl.flugplaetze-heliports.name')}</td>         <td>${c['attributes']['name'] or '-'}</td></tr>
     <tr><td class="cell-left">${_('ch.bazl.flugplaetze-heliports.location')}</td>           <td>${c['attributes']['location'] or '-'}</td></tr>
     <tr><td class="cell-left">${_('ch.bazl.flugplaetze-heliports.canton')}</td>     <td>${c['attributes']['canton'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bazl.flugplaetze-heliports.arp_east')}</td>         <td>${c['attributes']['arp_east'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bazl.flugplaetze-heliports.arp_north')}</td>         <td>${c['attributes']['arp_north'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.bazl.flugplaetze-heliports.elevation')}</td>         <td>${c['attributes']['elevation'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.bazl.flugplaetze-heliports.arp_east')}</td>         <td>${int(round(c['attributes']['arp_east'])) or '-'}</td></tr>
```

Comment:
Maybe it would be nice to add a custom filter (https://docs.makotemplates.org/en/latest/filtering.html?highlight=filter)

Reply:
@procrastinatio agree, but when searching the makos for round you will see, that it is being used all over: https://github.com/geoadmin/mf-chsdi3/search?l=Mako&p=3&q=round

please - have some pity with me ... but yes - the REST-Service will not round the results

Reply:
OK